.travis.yml

```diff
@@ -3,7 +3,7 @@ language: go
 go_import_path: go.uber.org/yarpc
 cache:
   directories:
-    - $HOME/.yarpc-go
+    - $HOME/.cache/yarpc-go
```

Comment:
How do you know that `XDG_CACHE_HOME` is either not set, or equals `${HOME}/.cache`? If it is not on travis, then this will fail. I'm betting this is just set on all linux distros, but I don't like assuming this. I'd prefer we make `CACHE_BASE` optionally set in `base.mk`, and explicitly set it as a global variable https://docs.travis-ci.com/user/environment-variables/#Global-Variables

```
env:
  global:
    - CACHE_BASE=$HOME/.cache/yarpc-go
```